src/internal/predicates.mjs

```diff
@@ -1,9 +1,15 @@
+import {FL} from './const';
+
+export function isAccessible(x){
+  return x !== null && x !== undefined;
```

Comment:
You could use `x != null` here. This is such a common idiom that I don't consider `isAccessible` warranted.